src/test/java/pl/project13/test/utils/AssertException.java

```diff
@@ -80,7 +80,7 @@ public void failWithExpectedButGot(@NotNull Class<? extends Throwable> expectedC
       }
     };
 
-    static interface Strategy {
+    interface Strategy {
```

Comment:
I was passing by it and wasn't sure to remove. @Test(expected = ...) does the job, switched to it.